### PR TITLE
WAVE2 1022 No html in character count

### PR DESCRIPTION
**Pull request by robbdavis**
WAVE2 1022 No html in character count

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module [hunk 1]
```diff
@@ -44,6 +44,15 @@ function ucgy_core_layout_form_alter(&$form, &$form_state, $form_id) {
     if (isset($form['#entity_type']) && $form['#entity_type'] == 'fieldable_panels_pane') {
         _ucgy_core_layout_update_colors($form);
     }
+
```
Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module]: do we need this commented out code?

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module [hunk 2]
```diff
@@ -44,6 +44,15 @@ function ucgy_core_layout_form_alter(&$form, &$form_state, $form_id) {
...
+//    if ($form['#id'] == 'fieldable-panels-panes-fieldable-panels-pane-content-type-edit-form'){
+//
+//     //sdpm($form);
+//      $path = drupal_get_path('module', 'ucgy_core_layout');
+//
+//      sdpm($path);
+//      drupal_add_js($path . '/js/maxlength.js');
+//    }
 }
```
Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module]: actually this one :)

Comment (robbdavis) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_layout/ucgy_core_layout.module]: I've taken it out. It left in by accident (part of testing for the maxlenght.js bug in the modal page edit form).

**Comment by balrajb** [comment]
can't see it working, @robbdavis screen share after scrum call?

**Pull request merged**